sched/wqueue/kwork_thread.c

```diff
@@ -239,7 +240,7 @@ static int work_thread(int argc, FAR char *argv[])
  ****************************************************************************/
 
 static int work_thread_create(FAR const char *name, int priority,
-                              int stack_size,
+                              FAR void *stack_addr, int stack_size,
```

Comment:
could we static tcb also? we have the internal change to create work thread from static resouce.
![image](https://github.com/user-attachments/assets/c5efbf3d-0f0b-426b-9c16-a7404d8504ab)

Reply:
The functionality of static configuration in tcb is also acceptable. You can submit your modifications for review.